components/Proposals/Proposal.jsx

```diff
@@ -225,7 +243,7 @@ const Proposal = ({ proposal, isAddressPresent }) => {
           <br />
           {!isExecutable && (
             <Text text="secondary">
-              {`To be executed, this proposal needs ${quorum} approvals. Current approvals: ${forVotes}`}
+              {`To be executed, this proposal needs ${remainingVeolasForApprovalInEth} veOLAS. Current veOLAS: ${totalVeOlasInEth}`}
```

Comment:
Here we can just replicate the component that I sketched in the Vercel comment. Here it is again for reference:
<img width="974" alt="image" src="https://github.com/valory-xyz/autonolas-contribution-service-frontend/assets/66292936/de4dab24-a0b9-4c52-b88f-930d6fd7439e">